### DISCUSSION TITLE
Annotations to 3d model having animation

### DISCUSSION THREAD

**Discussion by SOHDA31** [google/model-viewer#2803] [Q&A]
Can annotation feature be added to a 3d model which has animation? Like while the animation is happening the annotation isn't at any arbitary position in the enviroment but is placed on the 3d model and moving along with it? Any insights will be helpful and welcome. Thanks.

**Comment by elalish** [comment]
We don't have this feature yet, but we plan to add it. FYI @timmmeh

**Reply by SOHDA31** [reply]
Hey! thanks for the reply. Eagerly waiting for the updates. 
Thanks for everything you guys are doing.
